alpha/apps/kaltura/lib/storage/kStorageExporter.php

```diff
@@ -32,9 +32,9 @@ public function shouldConsumeChangedEvent(BaseObject $object, array $modifiedCol
 		// if changed object is entry
 		if($object instanceof entry && in_array(entryPeer::MODERATION_STATUS, $modifiedColumns) && $object->getModerationStatus() == entry::ENTRY_MODERATION_STATUS_APPROVED)
 			return true;
-		
-		// if changed object is flavor asset or thumb asset
-		if(($object instanceof flavorAsset || $object instanceof thumbAsset) && in_array(assetPeer::STATUS, $modifiedColumns) && $object->isLocalReadyStatus())
+
+		// if changed object is flavor asset / thumb asset / caption asset
+		if(($object instanceof flavorAsset || $object instanceof thumbAsset || $object instanceof captionAsset) && in_array(assetPeer::STATUS, $modifiedColumns) && $object->isLocalReadyStatus())
```

Comment:
Since lines of code repeats below - move to a function - shouldHandleAssetObject()
And check all the typtes and conditions inside reuse in #8574